### PR TITLE
get oracle image from Docker Hub

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: ci/build.yml [hunk 1]
```diff
@@ -214,12 +214,13 @@ jobs:
         eval "$(./dev-env/bin/dade-assist)"
         export ORACLE_PORT=1521
         export ORACLE_PWD=hunter2
-        docker login --username $ARTIFACTORY_USERNAME --password $ARTIFACTORY_PASSWORD digitalasset-daml-on-oracle-docker.jfrog.io
-        docker pull digitalasset-daml-on-oracle-docker.jfrog.io/oracle-xe-preloaded:2021-02-03-5a88b29
+        docker login --username "$DOCKER_LOGIN" --password "$DOCKER_PASSWORD" hub.docker.com
+        IMAGE=digitalasset/oracle-xe-18.4.0-preloaded:20210312-16-7009a64
```
Comment (cocreature) [inline ci/build.yml:218]: can we also switch to the non-xe version or do you want to do this separately?

**Review by cocreature — APPROVED** [review]
Thanks!

File: ci/build.yml (hunk 1 above)
Comment (garyverhaegen-da) [reply ci/build.yml:218]: Happy to switch; I guess I've not followed this closely enough to know we needed to switch.

Comment (cocreature) [reply ci/build.yml:218]: I _think_ oracle is reasonablly backwards compatible bu tthe other version is what will be run in production so seems better to test against that.

Comment (garyverhaegen-da) [reply ci/build.yml:218]: I did not manage to make it work, so I've reverted this PR to just doing the Docker Hub thing and I'll open a new one with "how far I got" in the switch to Oracle Entreprise.

**Pull request merged**